src/main/java/org/jitsi/videobridge/rest/JSONDeserializer.java

```diff
@@ -164,54 +165,85 @@ public static void deserializeCandidates(
             // lastN
             if (lastN != null)
             {
-                Integer i;
-
-                if (lastN instanceof Integer)
-                    i = (Integer) lastN;
-                else if (lastN instanceof Number)
-                    i = Integer.valueOf(((Number) lastN).intValue());
-                else
-                    i = Integer.valueOf(lastN.toString());
+                Integer i = objectToInteger(lastN);
                 channelIQ.setLastN(i);
```

Comment:
since you have the helper now, maybe it should just be `channelIQ.setLastN(objectToInteger(lastN))`?

Reply:
Fixed (all instances)